api/v1alpha1/sandbox_types.go

```diff
@@ -34,6 +44,17 @@ type SandboxSpec struct {
 
 // SandboxStatus defines the observed state of Sandbox.
 type SandboxStatus struct {
+	// FirstReadyTime - when did the sandbox become ready first
+	FirstReadyTime *metav1.Time `json:"firstReadyTime,omitempty"`
```

Comment:
Should this be under a `metav1.Conditions`, where there's a place for a timestamp? https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties

Reply:
Yes, the controller can start the TTL clock when a Ready condition is first seen.

Reply:
happy to use the ready condition timestamp instead.

Reply:
Gotcha, that makes sense. Can we add that later when we add TTL? The design might change, so they make sense to be added together.